app/models/micropost.rb

```diff
@@ -0,0 +1,12 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  has_one_attached :image
+  validates :user_id, presence: true
+  validates :content, presence: true, length: { maximum: Settings.validations.content.max_length }
+  validates :image, content_type: { in: Settings.image.type, message: I18n.t("errors.messages.invalid_image_format") }, size: { less_than: Settings.image.size.megabytes, message: I18n.t("errors.messages.invalid_image_size") }
+  scope :recent_posts, -> { order created_at: :desc }
```

Comment:
add 1 empty line before this line

Reply:
Here I have one extra ques wanted to ask:
![image](https://user-images.githubusercontent.com/68467813/92101447-b661bc00-ee0f-11ea-882d-4ee84010ed44.png)
For this problem, even i uploaded the image in jpeg or png format (VALID image format), but it still showed invalid. May i know how to solve it?

Reply:
you can change such as I suggested